faculty/clients/job.py

```diff
@@ -580,3 +580,19 @@ def get_subrun(
             project_id, job_id, run_identifier, subrun_identifier
         )
         return self._get(endpoint, SubrunSchema())
+
+    def cancel_run(self, project_id, job_id, run_identifier):
```

Comment:
I would change `run_identifier` to `run_id`, as this endpoint just accepts the run ID (a UUID), not either the run ID or run number (an int) as other endpoints do.

Reply:
Yeah, hope to work this out to be an actual `run_identifier` indeed, caught me in the act. :)

Reply:
Depends on https://bitbucket.org/theasi/steve/pull-requests/191